holoviews/core/spaces.py

```diff
@@ -788,6 +804,105 @@ def dynamic_redim(obj):
         return Dynamic(redimmed, shared_data=True, operation=dynamic_redim)
 
 
+    def collate(self):
+        """
+        Collation allows collapsing DynamicMaps with invalid nesting
+        hierarchies. This is particularly useful when defining
+        DynamicMaps returning an (Nd)Layout or GridSpace
+        type. Collating will split the DynamicMap into of individual
+        DynamicMaps. Note that the composite object has to be of
+        consistent length and types for this to work
+        correctly. Associating streams with specific viewables in the
+        returned container declare a stream_mapping on the DynamicMap
+        Callable during instantiation.
+        """
+        # Initialize
+        if self.last is not None:
+            pass
+        else:
+            self[self._initial_key()]
```

Comment:
Agreed. The reason I suggest the clone is so we don't need to worry about adding an option.